packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -147,22 +118,19 @@ function makeOperationGenerator(
 	}
 
 	// All subsequent helper functions are generators; note that they don't actually apply any operations.
-	function startPosition({ random, sharedString }: ClientOpState): number {
-		return random.integer(0, Math.max(0, sharedString.getLength() - 1));
+	function startPosition({ random, channel }: ClientOpState): number {
```

Comment:
Why the change in name to "channel" since it's not used like a channel?

Reply:
the prop comes from the generic infrastructure, so it shouldn't be `sharedString`. we could rename the local to `sharedString` if we want. naming the field `dds` would also be reasonable and I considered it. if you think that's better I'm open to changing